### PR TITLE
login to container-registry on ci commands and bump go test timeout

### PR DESCRIPTION
**Pull request by alecholmez**
Signed-off-by: Alec Holmes <alecholmez@me.com>

### CODE REVIEW

**Review by dontlaugh — DISMISSED** [review]
Put these calls internal to cibuild, where they're required.

File: scripts/cibuild [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 set -e
 
+cmd_login () {
+  # ci needs to login to the container registries since we mirror 
+  # a golang builder image to circumvent dockerhub rate limiting.
+  container-registry-login "$BUILDKITE_ARTIFACTORY_USER" "$BUILDKITE_ARTIFACTORY_PASSWORD" greymatter.jfrog.io
+  container-registry-login "$QUAY_USERNAME" "$QUAY_PASSWORD" quay.io
+}
+
```
Comment (dontlaugh) [inline scripts/cibuild:11]: Dont' add a function for this; copy the calls to `container-registry-login` close to _exactly_ where they're needed

Comment (dontlaugh) [reply scripts/cibuild:11]: Eventually we'll get rid of quay; the verbosity is a signal to us to fix this in the future.

File: .buildkite/pipeline-fips.yaml [hunk 2]
```diff
@@ -3,4 +3,5 @@ agents:
 steps:
   - label: ":shipit: Build FIPS"
     commands:
+      - ./scripts/cibuild login
       - ./scripts/cibuild build
```
Comment (dontlaugh) [inline .buildkite/pipeline-fips.yaml:7]: Add a newline here

**Review by danielpcox — DISMISSED** [review]
Gracias

**Review by danielpcox — APPROVED** [review]
Thoroughly approved

**Pull request merged**